### PR TITLE
Upgrade ci to support Python 3

### PR DESCRIPTION
**Pull request by ofek**
### Motivation

https://github.com/DataDog/integrations-core/pull/3243

Also fixed flake8 as was done in ^

### CODE REVIEW

File: reboot_required/tox.ini [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py37
 envlist =
-    reboot_required
+    py27-reboot_required
     flake8
 
 [testenv]
```
Comment (gzussa) [inline reboot_required/tox.ini]: let's add `-v` to `pytest`

Comment (gzussa) [reply reboot_required/tox.ini]: Also let's add `platform = linux|darwin|win32` for `[flake8]`

Comment (gzussa) [reply reboot_required/tox.ini]: `exclude = .eggs,.tox,build` ?

File: eventstore/tox.ini [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
+envlist =
+    py27-{unit,integration}
+    flake8
 
 [testenv]
 platform = linux|darwin
 deps =
     datadog-checks-base[deps]
     -rrequirements-dev.txt
-
```
Comment (gzussa) [inline eventstore/tox.ini]: 👍

File: eventstore/tox.ini [hunk 3]
```diff
@@ -1,23 +1,19 @@
 [tox]
 minversion = 2.0
-basepython = py27
-envlist = unit, integration, flake8
+basepython = py37
+envlist =
+    py27-{unit,integration}
```
Comment (gzussa) [inline eventstore/tox.ini]: `py{27,37}`?

File: reboot_required/tox.ini [hunk 4]
```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py37
 envlist =
-    reboot_required
+    py27-reboot_required
```
Comment (gzussa) [inline reboot_required/tox.ini]: `py{27,37}`?

File: sortdb/tox.ini [hunk 5]
```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py37
 envlist =
-    sortdb
+    py27-sortdb
     flake8
 
 [testenv]
```
Comment (gzussa) [inline sortdb/tox.ini]: `exclude = .eggs,.tox,build` and `platform = linux|darwin|win32` for `[flake8]`

File: sortdb/tox.ini [hunk 6]
```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py37
 envlist =
-    sortdb
+    py27-sortdb
```
Comment (gzussa) [inline sortdb/tox.ini]: `py{27,37}`

File: traefik/tox.ini [hunk 7]
```diff
@@ -1,26 +1,19 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py37
 envlist =
-    unit,
-    integration,
+    py27-{unit,integration}
```
Comment (gzussa) [inline traefik/tox.ini]: same as above?

File: traefik/tox.ini [hunk 8]
```diff
@@ -1,26 +1,19 @@
...
     flake8
 
 [testenv]
 platform = linux|darwin
```
Comment (gzussa) [inline traefik/tox.ini]: let's run flake on win32 as well here

Comment (gzussa) [reply traefik/tox.ini]: and let's exclude `build` as well

**Comment by ofek** [comment]
I'll address individual Python 3 & platform compatibility in separate PRs

**Pull request merged**